tst/testinstall/ConjNatSym.tst

```diff
@@ -0,0 +1,14 @@
+# The following two test use the new version for natural symmetric group
```

Comment:
"the new version" of what/? Just looking at this file doesn't make that clear. How about something more like this:
> This file tests an`IsConjugate` method for natural symmetric groups, and its various cases.